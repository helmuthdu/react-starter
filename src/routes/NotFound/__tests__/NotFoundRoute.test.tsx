import { shallow } from 'enzyme';
import React from 'react';
import NotFoundRoute from '../NotFoundRoute';

describe('Route -> NotFound', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<NotFoundRoute />);
    expect(wrapper).toMatchSnapshot();
  });
});
