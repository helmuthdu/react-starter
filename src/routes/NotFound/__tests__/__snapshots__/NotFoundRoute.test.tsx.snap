// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Route -> NotFound should match snapshot 1`] = `
<div>
  <h1>
    You are here!
  </h1>
  <h2>
    But nothing found for you #404
  </h2>
</div>
`;
